cli/tests/upgrade.bats

```diff
@@ -173,106 +175,92 @@ EOF
-  assert_equal "$old_hello_locked_rev" "$old_hello_response_rev"
+  old_hello_response_drv=$(jq -r '.[0].[0].page.packages[0].derivation' "$GENERATED_DATA/resolve/old_hello.json")
+  old_hello_locked_drv=$(jq -r '.packages.[0].derivation' "$LOCK_PATH")
+  assert_equal "$old_hello_locked_drv" "$old_hello_response_drv"
 
   _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/hello.json" \
     run "$FLOX_BIN" upgrade toplevel
   assert_output --partial "Upgraded 'hello'"
-  hello_response_rev=$(jq -r '.[0].[0].pages.[0].page' "$GENERATED_DATA/resolve/hello.json")
-  hello_locked_rev=$(jq -r '.packages.[0].rev_count' "$LOCK_PATH")
-  assert_equal "$hello_locked_rev" "$hello_response_rev"
+  hello_response_drv=$(jq -r '.[0].[0].page.packages[0].derivation' "$GENERATED_DATA/resolve/hello.json")
+  hello_locked_drv=$(jq -r '.packages.[0].derivation' "$LOCK_PATH")
+  assert_equal "$hello_locked_drv" "$hello_response_drv"
   
-  assert_not_equal "$old_hello_locked_rev" "$hello_locked_rev"
+  assert_not_equal "$old_hello_locked_drv" "$hello_locked_drv"
 }
 
 @test "catalog: upgrade by iid" {
-  skip "FIXME: upgrades package but reports that no packages were upgraded"
-
   "$FLOX_BIN" init
   _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/old_hello.json" "$FLOX_BIN" install hello
 
-  old_hello_response_rev=$(jq -r '.[0].[0].pages.[0].page' "$GENERATED_DATA/resolve/old_hello.json")
-  old_hello_locked_rev=$(jq -r '.packages.[0].rev_count' "$LOCK_PATH")
-  assert_equal "$old_hello_locked_rev" "$old_hello_response_rev"
+  old_hello_response_drv=$(jq -r '.[0].[0].page.packages[0].derivation' "$GENERATED_DATA/resolve/old_hello.json")
+  old_hello_locked_drv=$(jq -r '.packages.[0].derivation' "$LOCK_PATH")
+  assert_equal "$old_hello_locked_drv" "$old_hello_response_drv"
 
   _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/hello.json" \
     run "$FLOX_BIN" upgrade hello
   assert_output --partial "Upgraded 'hello'"
-  hello_response_rev=$(jq -r '.[0].[0].pages.[0].page' "$GENERATED_DATA/resolve/hello.json")
-  hello_locked_rev=$(jq -r '.packages.[0].rev_count' "$LOCK_PATH")
-  assert_equal "$hello_locked_rev" "$hello_response_rev"
+  hello_response_drv=$(jq -r '.[0].[0].page.packages[0].derivation' "$GENERATED_DATA/resolve/hello.json")
+  hello_locked_drv=$(jq -r '.packages.[0].derivation' "$LOCK_PATH")
+  assert_equal "$hello_locked_drv" "$hello_response_drv"
   
-  assert_not_equal "$old_hello_locked_rev" "$hello_locked_rev"
+  assert_not_equal "$old_hello_locked_drv" "$hello_locked_drv"
 }
 
 @test "catalog: upgrade errors on iid in group with other packages" {
-  skip "FIXME: upgrades package but reports that no packages were upgraded"
-
   "$FLOX_BIN" init
   _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/curl_hello.json" "$FLOX_BIN" install curl hello
 
-  export _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/hello.json"
-  run "$FLOX_BIN" upgrade hello
+  _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/hello.json" \
```

Comment:
Nit: this response would be more accurate but it's a smell of our mocks that it doesn't matter:
```suggestion
  _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/curl_hello.json" \
```

Reply:
Actually, it doesn't matter because this test never hits the client at all, so really this could be `empty.json`